### PR TITLE
2 packages from yasuo-ozu/satysfi-latexcmds at 0.1.1

### PR DESCRIPTION
**Pull request by yasuo-ozu**
# Automatic follow-ups
Choose follow-up actions.  Do not write anything after this section.
- [x] Add to snapshot `snapshot-develop` :: satysfi-latexcmds-doc.0.1.1 satysfi-latexcmds.0.1.1
- [x] Add to snapshot `snapshot-develop--1` :: satysfi-latexcmds-doc.0.1.1 satysfi-latexcmds.0.1.1
- ~~Add to snapshot `snapshot-stable-0-0-4`~~ (Inconsistent)
- ~~Add to snapshot `snapshot-stable-0-0-5`~~ (Inconsistent)
- [x] Add to snapshot `snapshot-stable-0-0-6` :: satysfi-latexcmds-doc.0.1.1 satysfi-latexcmds.0.1.1
- [x] Add to snapshot `snapshot-stable-0-0-6--1` :: satysfi-latexcmds-doc.0.1.1 satysfi-latexcmds.0.1.1
- [x] Add to snapshot `snapshot-stable-0-0-7` :: satysfi-latexcmds-doc.0.1.1 satysfi-latexcmds.0.1.1

### CODE REVIEW

**Comment by yasuo-ozu** [comment]
satysfi <0.1.0 を satysfi <0.1 に修正するために force-push しました。

**Comment by na4zagin3** [comment]
@yasuo-ozu Apparently, the doc library does not support SATySFi 0.0.5.  Let's drop SATySFi 0.0.5 support. 

[Build (ubuntu-latest, 4.11.1, stable-0-0-5)](https://github.com/na4zagin3/satyrographos-repo/runs/6628265606?check_suite_focus=true#logs)

```
#  ---- ---- ---- ----
#   target file: 'latexcmds-doc.pdf'
#   dump file: 'latexcmds-doc.satysfi-aux' (will be created)
#   parsing 'latexcmds-doc.saty' ...
# ! [Syntax Error at Parser] at "latexcmds-doc.saty", line 320, characters 101-103:
# Reading runtime dist: /home/runner/work/satyrographos-repo/satyrographos-repo/_opam/share/satysfi/dist
```

```
% latexcmds-doc.saty:320
         +p {とすれば${\overbrace?:{\bigcirc\ ..\ \bigcirc}?:!(Color.red)?:!(1.0){\underbrace?*?:!(Color.red){-------}}}となる。}
```

File: packages/satysfi-latexcmds/satysfi-latexcmds.0.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+maintainer: "Yasuo Ozu <yasuo@ozu.email>"
+authors: "Yasuo Ozu <yasuo@ozu.email>"
+license: "LGPL-3.0-or-later"
+homepage: "https://github.com/yasuo-ozu/satysfi-latexcmds"
+bug-reports: "https://github.com/yasuo-ozu/satysfi-latexcmds/issues"
+dev-repo: "git+https://github.com/yasuo-ozu/satysfi-latexcmds.git"
+
+depends: [
+  "satysfi" {>= "0.0.5" & < "0.1"}
```
Comment (na4zagin3) [inline packages/satysfi-latexcmds/satysfi-latexcmds.0.1.1/opam:13]:
```suggestion
  "satysfi" {>= "0.0.6" & < "0.1"}
```

File: packages/satysfi-latexcmds-doc/satysfi-latexcmds-doc.0.1.1/opam [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+maintainer: "Yasuo Ozu <yasuo@ozu.email>"
+authors: "Yasuo Ozu <yasuo@ozu.email>"
+license: "LGPL-3.0-or-later"
+homepage: "https://github.com/yasuo-ozu/satysfi-latexcmds"
+bug-reports: "https://github.com/yasuo-ozu/satysfi-latexcmds/issues"
+dev-repo: "git+https://github.com/yasuo-ozu/satysfi-latexcmds.git"
+
+depends: [
+  "satysfi" {>= "0.0.5" & < "0.1"}
```
Comment (na4zagin3) [inline packages/satysfi-latexcmds-doc/satysfi-latexcmds-doc.0.1.1/opam:13]:
```suggestion
  "satysfi" {>= "0.0.6" & < "0.1"}
```

**Comment by na4zagin3** [comment]
Merging.  Thank you!

File: packages/satysfi-latexcmds/satysfi-latexcmds.0.1.1/opam (hunk 1 above)
Comment (yasuo-ozu) [reply packages/satysfi-latexcmds/satysfi-latexcmds.0.1.1/opam]: Thank you. I will apply it to upstream.

**Pull request merged**